lib/cc/services/github_pull_requests.rb

```diff
@@ -72,19 +57,27 @@ def simple_failure(message)
     { ok: false, message: message }
   end
 
+  def response
+    @response || simple_failure("Nothing happened")
+  end
+
   def update_status_skipped
-    update_status("success", "Code Climate has skipped analysis of this commit.")
+    update_status(
+      "success",
+      "Code Climate has skipped analysis of this commit."
+    )
   end
 
   def update_status_success
     update_status("success", "Code Climate has analyzed this pull request.")
+    add_comment
```

Comment:
That's fair. Perhaps a separate PR then is appropriate for this change. I think it's in scope however, because we're now capturing the result of these calls -- and the most important result is the result of posting a status to a PR (not the comment).